### ISSUE TITLE
Android/Cordova UiRippleInk error

### ISSUE THREAD

**Issue by stbaer** [JosephusPaye/Keen-UI#144]
https://github.com/JosephusPaye/Keen-UI/blob/master/src/UiRippleInk.vue#L15

The startRipple method throws an error because event.currentTarget is undefined.
![screenshot from 2016-09-10 13-10-20](https://cloud.githubusercontent.com/assets/4010327/18408139/3a01f1b0-7759-11e6-9dd2-14e582c1bdf2.png)

**Comment by JosephusPaye** [comment]
Yeah that a bug, probably should check for `event.currentTarget` and return early if `false`.

**Comment by stbaer** [comment]
Ya, the check and return is already there, L20-22. Moving it after setting the holder variable should be enough.

**Comment by stbaer** [comment]
Hi @JosephusPaye, would you consider this for hotfix before the next release?

**Comment by fenivana** [comment]
The bug is caused by `touchstart` event. the `Touch` object (https://developer.mozilla.org/en-US/docs/Web/API/Touch) doesn't have `currentTarget` property. You should either remove the `handleTouchStart` event handler completely or make it right, rather than leaves the buggy `handleTouchStart` there to do a useless `for` loop.

```js
var handleTouchStart = function handleTouchStart(e) {
    if (e.changedTouches) {
        for (var i = 0; i < e.changedTouches.length; ++i) {
            startRipple(e.type, e.changedTouches[i]);
        }
    }
};
```

**Comment by JosephusPaye** [comment]
Fixed in v1.0.

**Comment by brickgale** [comment]
is there by any chance we can put the fix on v0.8.9? 
Here is my PR: https://github.com/JosephusPaye/Keen-UI/pull/280
